contrib/grinder/scripts/utils.py

```diff
@@ -177,8 +177,8 @@ def check_position(self, logger, max_position):
 
   @classmethod
   def time(cls):
-    return time.time()
+    return int(time.time() * 1000)
 
   @classmethod
   def sleep(cls, x):
-    return time.sleep(x)
+    return time.sleep(x/1000)
```

Comment:
why are we multiplying by 1000 https://github.com/rackerlabs/blueflood/pull/501/files#diff-413f7f2bf24ab7bf69b97eda82cdd766R66 and dividing by 1000 here?